.buildbot.sh

```diff
@@ -126,5 +126,5 @@ args=(
       )
 BUILDBOT_PLATFORM=morello-purecap python3 ci/run_cheri_bdwgc_tests.py "${args[@]}"
 
-echo "removing up unit-test files"
+echo "removing ${BUILDBOT_PLATFORM} unit-test files"
```

Comment:
Isn't `BUILDBOT_PLATFORM` only defined for the respective `ci/run_cheri_bdwgc_tests.py` calls?

Reply:
Thanks - another slip up caught. Fixed in [f6227c7](https://github.com/capablevms/bdwgc/pull/17/commits/f6227c7334a5a1537f5b020c205da80282a3b59c)